nilearn/_utils/docs.py

```diff
@@ -3,310 +3,195 @@
+    is no surface data (either because ``surf_data`` contains ``nan``\s
+    or because is was thresholded).
 
-    Default=None."""
+        .. note::
+            This non-uniformly changes the surf_data values according
+            to e.g the sulcal depth.
 
-# Low_pass
-docdict['low_pass'] = """
-low_pass : :obj:`float` or None, optional
-    Low cutoff frequency in Hertz. If specified, signals above this
-    frequency will be filtered out. If None, no low-pass filtering
-    will be performed. Default=None."""
+"""
 
-# lower_cutoff
-docdict['lower_cutoff'] = """
-lower_cutoff : :obj:`float`, optional
-    Lower fraction of the histogram to be discarded."""
+# bg_vmax
+docdict['bg_vmax'] = """
+bg_vmin : :obj:`float`, optional
+    vmax for ``bg_img``."""
 
-# upper_cutoff
-docdict['upper_cutoff'] = """
-upper_cutoff : :obj:`float`, optional
-    Upper fraction of the histogram to be discarded."""
+# bg_vmin
+docdict['bg_vmin'] = """
+bg_vmin : :obj:`float`, optional
+    vmin for ``bg_img``."""
 
-# connected
-docdict['connected'] = """
-connected : :obj:`bool`, optional
-    If connected is True, only the largest connect component is kept."""
+# black_bg
+docdict['black_bg'] = """
+black_bg : :obj:`bool`, or 'auto', optional
+    If ``True``, the background of the image is set to be black.
+    If you wish to save figures with a black background, you
+    will need to pass facecolor='k', edgecolor='k'
+    to :func:`matplotlib.pyplot.savefig`."""
 
 # border_size
 docdict['border_size'] = """
 border_size : :obj:`int`, optional
     The size, in :term:`voxel` of the border used on the side of
     the image to determine the value of the background."""
 
-# opening
-docdict['opening'] = """
-opening : :obj:`bool` or :obj:`int`, optional
-    This parameter determines whether a morphological
-    :term:`opening<Opening>` is performed, to keep only large structures.
-    This step is useful to remove parts of the skull that might have been
-    included. ``opening`` can be:
+# cbar_tick_format
+docdict['cbar_tick_format'] = """
+cbar_tick_format : :obj:`str`, optional
+    Controls how to format the tick labels of the colorbar.
+    Ex: use "%%.2g" to display using scientific notation."""
 
-        - A boolean : If False, no :term:`opening<Opening>` is performed.
-          If True, it is equivalent to ``opening=1``.
-        - An integer `n`: The :term:`opening<Opening>` is performed via `n`
-          :term:`erosions<Erosion>` (see :func:`scipy.ndimage.binary_erosion`).
-          The largest connected component is then estimated if ``connected`` is
-          set to True, and 2`n` :term:`dilation<Dilation>` operations are
-          performed (see :func:`scipy.ndimage.binary_dilation`) followed by
-          `n` :term:`erosions<Erosion>`. This corresponds to 1
-          :term:`opening<Opening>` operation of order `n` followed by a
-          :term:`closing<Closing>` operator of order `n`.
+# classifier_options
```

Comment:
You can use intersphinx linking here and get rid of `base_url` and `SKLEARN_LINKS`. See example below

Reply:
should be done

Reply:
Great thx! Looks right and links work in the doc artifacts

Reply:
will wait for CI to complete and will merge then